modules/claims_api/spec/models/veteran/service/user_spec.rb

```diff
@@ -0,0 +1,55 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+describe Veteran::User do
+  context 'initialization' do
+    let(:user) do
+      ClaimsApi::Veteran.new(
+        uuid: '123456789',
+        ssn: '123456789',
+        first_name: 'Firstname',
+        last_name: 'Lastname',
+        va_profile: ClaimsApi::Veteran.build_profile('1970-01-01'),
+        last_signed_in: Time.now.utc
+      )
+    end
```

Comment:
i pretty much just copied the test from here:
https://github.com/department-of-veterans-affairs/vets-api/blob/master/modules/veteran/spec/models/veteran/user_spec.rb#L7

but send in our ClaimsApi::Veteran instance instead.

i was able to recreate the undefined method error that way and then see it resolve when fixed.